apps/graphql/src/dataloaders/Itinerariesloader.js

```diff
@@ -21,17 +21,51 @@ export type ItinerariesSearchParameters = {|
   |},
 |};
 
-export type Itineraries = {|
+export type TripSector = {
+  +city: string,
+  +cityCode: string,
+  +localTime: Date,
+  +utcTime: Date,
+};
+export type RouteItem = {
+  +airline: string,
+  +arrival: TripSector,
+  +departure: TripSector,
   +id: string,
+};
+
+export type Price = {
+  +amount: number,
+  +currency: string,
+};
+
+export type Itineraries = {|
   +airlines: Array<string>,
-  +price: number,
+  +id: string,
   +flyFrom: string,
   +flyTo: string,
   +localDeparture: string,
   +localArrival: string,
+  +price: Price,
+  +route: Array<RouteItem>,
+  +routes: Array<Array<string>>,
 |};
 
+export type ApiRouteItem = {
+  +airline: string,
```

Comment:
Aren't we assuming here that the API response will never give null or undefined?

Reply:
OK, fixed